docs/changes/newsfragments/270.enh

```diff
@@ -0,0 +1 @@
+Remove final model fit requirement for inspector to be returned by `run_cross_validation` by `Fede Raimondo`_.
```

Comment:
`run_cross_validation` needs to be either:
```
``..``
```
or cross-refd using:
```
:func:`...`
```
else it'll be in italics.

Reply:
done